jelixir/lib/jelixir.ex

```diff
@@ -35,8 +107,11 @@ defmodule JelixirLib do
 
   def create_gen(name, node_result) do
     schema_name = String.downcase(name)
-    capitalize_module_name = schema_name |> String.capitalize()
+    module_name = schema_name |> String.capitalize()
```

Comment:
@ttpho I think it should be `module_name = String.capitalize(schema_name)` because it should be the same with `schema_name = String.downcase(name)`